src/main/java/HW_7/Circle.java

```diff
@@ -0,0 +1,23 @@
+package HW_7;
+
+public class Circle extends Figure {
+    private double radius;
+    public static final double PI = 3.14;
+    public Circle(double radius){
+        this.radius = radius;
+    }
+
+    @Override
+    public double getPerimeter() {
+        return 2 * PI * radius;
+    }
+
+    @Override
+    public double getArea() {
+        return PI * radius * radius;
+    }
```

Comment:
there is should an extra line between 18 and 19 line

Reply:
fixed

Reply:
not fixed